bitnami/clickhouse/templates/keeper/usersd-configmap.yaml

```diff
@@ -0,0 +1,23 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if and .Values.keeper.enabled .Values.keeper.extraUsersFiles (not .Values.keeper.existingExtraUsersConfigmap) }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "clickhouse.keeper.usersd.configmapName" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" (dict "customLabels" .Values.commonLabels "context" .) | nindent 4 }}
+    app.kubernetes.io/component: keeper
+    app.kubernetes.io/part-of: clickhouse
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" (dict "value" .Values.commonAnnotations "context" .) | nindent 4 }}
+  {{- end }}
+data:
+  {{- range $key, $value := .Values.keeper.extraUsersFiles }}
+  {{ $key }}: |
+    {{- include "common.tplvalues.render" (dict "value" $value "context" $) | nindent 4 }}
+  {{- end }}
```

Comment:
I do not recall if the users file contained passwords. In the past we used a configmap because it was dependant on an env var, but now maybe it makes sense to be a secret

Reply:
I see that you are mimicking what we were already doing. As it is a major change, could you double check?

Reply:
I added support for loading `users.d` config files from secrets at https://github.com/bitnami/charts/pull/32958/commits/74277fc4228d71b6770e137585dded88f8ac9514